### PR TITLE
Vkanithi

### PR DESCRIPTION
**Pull request by vkanithi**

### CODE REVIEW

**Comment by vkanithi** [comment]
Tested on integration

File: _slides/10-vkanithi.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Let's have a coffee break
```
Comment (MarlaChat) [inline _slides/10-vkanithi.md:6]: 👍

Comment (MarlaChat) [inline _slides/10-vkanithi.md:6]:
```suggestion
Let's have a snack break
```

**Review by MarlaChat — APPROVED** [review]

File: _slides/10-vkanithi.md (hunk 1 above)
Comment (vkanithi) [reply _slides/10-vkanithi.md:6]: The pic shows Coffee, so Coffee break is better

Comment (MarlaChat) [reply _slides/10-vkanithi.md:6]: That is fine.  I just don't drink coffee.

Comment (MarlaChat) [reply _slides/10-vkanithi.md:6]: Sounds good

**Review by MarlaChat — APPROVED** [review]

**Pull request merged**